response_operations_ui/views/collection_exercise.py

```diff
@@ -568,7 +568,9 @@ def get_create_collection_exercise_form(survey_ref, short_name):
     logger.info("Retrieving survey data for form", short_name=short_name, survey_ref=survey_ref)
     form = CreateCollectionExerciseDetailsForm(form=request.form)
     survey_details = survey_controllers.get_survey(short_name)
-    survey_eq_version = survey_details["eqVersion"] if survey_details["surveyMode"] != "SEFT" else ""
+    survey_eq_version = "v2"
```

Comment:
Can you put a comment that this is here in the unlikely event the eqVersion field is blank in the survey?